api/valor_api/backend/metrics/classification.py

```diff
@@ -313,31 +559,31 @@ def _compute_curves(
                     },
                 }
 
-    output = []
-
-    output.append(
+    pr_curves = [
         schemas.PrecisionRecallCurve(
-            label_key=grouper_key, value=dict(pr_output)
-        ),
-    )
+            label_key=label_key,
+            value=dict(value),
+        )
+        for label_key, value in pr_output.items()
+    ]
 
-    if enums.MetricType.DetailedPrecisionRecallCurve in metrics_to_return:
-        output += [
-            schemas.DetailedPrecisionRecallCurve(
-                label_key=grouper_key, value=dict(detailed_pr_output)
-            )
-        ]
+    detailed_pr_curves = [
```

Comment:
are we making it so we always run DetailedPRCurves by removing the wold `if` condition on 324? not sure I'm reading this change correctly.

Reply:
This is just a result aggregator, since `detailed_pr_output` is empty this list comprehension will be skipped.

Reply:
The computation block is earlier in this function and should be controlled by an `if` statement

Reply:
See line 451!